doc/v2/documentation_data_format.rst

```diff
@@ -651,53 +651,53 @@ Detailed field description
   observable formula contains an override, and a proportional noise model is
   used, which means the observable formula also appears in the noise formula.
 
-- ``noiseDistribution`` [STRING: 'normal' or 'laplace', OPTIONAL]
+* ``noiseDistribution`` [STRING, OPTIONAL]
 
-  Assumed noise distribution for the given measurement. Only normally or
-  Laplace distributed noise is currently allowed (log-normal and
-  log-Laplace are obtained by setting ``observableTransformation`` to ``log``, similarly for ``log10``).
-  Defaults to ``normal``. If ``normal``, the specified ``noiseParameters`` will be
-  interpreted as standard deviation (*not* variance). If ``Laplace`` ist specified, the specified ``noiseParameter`` will be interpreted as the scale, or diversity, parameter.
+  Assumed noise distribution for the measurements of the given observable.
+  The supported :ref:`noise distributions <v2_noise_distributions>` and the
+  respective interpretation of ``noiseFormula`` are given in the table below.
 
-.. _noise_distributions:
+.. _v2_noise_distributions:
 
 Noise distributions
 ~~~~~~~~~~~~~~~~~~~
 
-For ``noiseDistribution``, ``normal`` and ``laplace`` are supported. For ``observableTransformation``, ``lin``, ``log`` and ``log10`` are supported. Denote by :math:`y` the simulation, :math:`m` the measurement, and :math:`\sigma` the standard deviation of a normal, or the scale parameter of a laplace model, as given via the ``noiseFormula`` field. Then we have the following effective noise distributions.
-
-- Normal distribution:
-
-  .. math::
-     \pi(m|y,\sigma) = \frac{1}{\sqrt{2\pi}\sigma}\exp\left(-\frac{(m-y)^2}{2\sigma^2}\right)
-
-- Log-normal distribution (i.e. log(m) is normally distributed):
-
-  .. math::
-     \pi(m|y,\sigma) = \frac{1}{\sqrt{2\pi}\sigma m}\exp\left(-\frac{(\log m - \log y)^2}{2\sigma^2}\right)
-
-- Log10-normal distribution (i.e. log10(m) is normally distributed):
-
-  .. math::
-     \pi(m|y,\sigma) = \frac{1}{\sqrt{2\pi}\sigma m \log(10)}\exp\left(-\frac{(\log_{10} m - \log_{10} y)^2}{2\sigma^2}\right)
-
-- Laplace distribution:
-
-  .. math::
-     \pi(m|y,\sigma) = \frac{1}{2\sigma}\exp\left(-\frac{|m-y|}{\sigma}\right)
+Denote by :math:`m` the measurement, :math:`y` the simulation
```

Comment:
Currently I think there is nothing in this document (except the changelog) that states that `observableFormula` defines the location. Could be in the table column definition/description and/or here

```suggestion
Denote by :math:`m` the measurement, :math:`y:=\text{observableFormula}` the simulation
```

Reply:
Calling $y$ simulation here and after the table is fine but might be confusing since we also use simulation to refer to things other than a single observable at a single timepoint for a single experimental condition. e.g. in PEtab a simulation might also be the task and all output from solving the ODE for a single experimental condition. "The simulated value" might be clearer but fine as is

Reply:
Updated.